src/modules/NtpClient.rb

```diff
@@ -812,13 +812,13 @@ def update_cron_settings
       if @synchronize_time
         SCR.Write(
           path(".target.string"),
-          @cron_file,
+          CRON_FILE,
           "-*/#{@sync_interval} * * * * root /usr/sbin/chronyd -q &>/dev/null\n"
         )
       else
         SCR.Execute(
           path(".target.bash"),
-          "test -e #{@cron_file} && rm #{@cron_file};"
+          "test -e #{CRON_FILE} && rm #{CRON_FILE};"
```

Comment:
NP: there is a trailing `;` and maybe simple `rm -f #{CRON_FILE}` would be better (even could be replaced by native Ruby)